deeppavlov/utils/server/server.py

```diff
@@ -100,6 +108,15 @@ def interact(model: Chainer, params_names: List[str]) -> Tuple[Response, int]:
     return jsonify(result), 200
 
 
+def test_interact(model: Chainer, params_names: List[str]) -> Tuple[Response, int]:
+    model_args = ["Test string." for _ in params_names]
```

Comment:
Model takes batches as arguments, so
```suggestion
    model_args = [["Test string."] for _ in params_names]
```
Also, are you sure that all the inputs will always be texts?